packages/expo/src/SQLite.ts

```diff
@@ -3,8 +3,15 @@ import './timer/polyfillNextTick';
 import zipObject from 'lodash.zipobject';
 import { NativeModules, Platform } from 'react-native';
 import customOpenDatabase from '@expo/websql/custom';
```

Comment:
We don't need to pull this in for the web -- it's a WebSQL polyfill for Node. This is a rare case where it makes sense to have SQLite.ts and SQLite.web.ts, which just exports `{ openDatabase: window.openDatabase }`